### PR TITLE
Add version command for tkctl

### PR DESCRIPTION
**Pull request by aylei**
This is related to #452 

Signed-off-by: Aylei <rayingecho@gmail.com>

### What problem does this PR solve?
Add version command for `tkctl`
### What is changed and how it works?
New go file
### Check List
Manual tested:
```shell
$ tkctl version
Client Version: v1.0.0-beta.1-p2-103-g84a9098b20114c-dirty
TiDB Controller Manager Version: pingcap/tidb-operator:latest
TiDB Scheduler Version:
	tidb-scheduler: pingcap/tidb-operator:latest
	kube-scheduler: uhub.ucloud.cn/pingcap/hyperkube:v1.10.2_coreos.0

$ tkctl version -c
Client Version: v1.0.0-beta.1-p2-103-g84a9098b20114c-dirty
```

### CODE REVIEW

File: pkg/tkctl/cmd/version/version.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	if err != nil {
+		return err
+	}
+
+	clientVersion := version.Get()
+
+	controllers, err := kubeCli.AppsV1().
+		Deployments(core.NamespaceAll).
+		List(v1.ListOptions{
+			LabelSelector: "app.kubernetes.io/component=controller-manager,app.kubernetes.io/instance=tidb-operator",
```
Comment (weekface) [inline pkg/tkctl/cmd/version/version.go]: use `label. ComponentLabelKey` instead.

File: pkg/tkctl/cmd/version/version.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+		})
+	if err != nil {
+		return err
+	}
+	schedulers, err := kubeCli.AppsV1().
+		Deployments(core.NamespaceAll).
+		List(v1.ListOptions{
+			LabelSelector: "app.kubernetes.io/component=scheduler,app.kubernetes.io/instance=tidb-operator",
```
Comment (weekface) [inline pkg/tkctl/cmd/version/version.go]: ditto

**Comment by tennix** [comment]
Can we show the kube-scheduler version bundled with tidb-scheduler, this may be useful when troubleshooting.

File: pkg/tkctl/cmd/version/version.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+			fmt.Fprintf(o, "\t%s: %s\n", item.Name, item.Spec.Template.Spec.Containers[0].Image)
+		}
+	}
+	if len(schedulers.Items) == 0 {
+		fmt.Fprintf(o, "No TiDB Scheduler found, please install one first\n")
+	} else if len(schedulers.Items) == 1 {
+		fmt.Fprintf(o, "TiDB Scheduler Version: %s\n", schedulers.Items[0].Spec.Template.Spec.Containers[0].Image)
+	} else {
+		// warn for multiple scheduler
+		fmt.Fprintf(o, "WARN: more than one TiDB Scheduler instance found, this is un-supported and may lead to un-specified behavior:\n")
```
Comment (onlymellb) [inline pkg/tkctl/cmd/version/version.go]:
```suggestion
		fmt.Fprintf(o, "WARN: more than one TiDB Scheduler instance found, this is un-supported and may lead to un-expected behavior:\n")
```

File: pkg/tkctl/cmd/version/version.go [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+		Writer: out,
+	}
+	cmd := &cobra.Command{
+		Use:     "version",
+		Short:   "Print the client & server version",
+		Example: versionExample,
+		Run: func(_ *cobra.Command, _ []string) {
+			cmdutil.CheckErr(options.runVersion(tkcContext))
+		},
+	}
```
Comment (onlymellb) [inline pkg/tkctl/cmd/version/version.go]: Should add an options `-c,--client` to show client version only, which will be useful without server environment.

File: pkg/tkctl/cmd/version/version.go [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+	schedulers, err := kubeCli.AppsV1().
+		Deployments(core.NamespaceAll).
+		List(v1.ListOptions{
+			LabelSelector: fmt.Sprintf("%s=%s,%s=%s", label.ComponentLabelKey, "scheduler", label.InstanceLabelKey, "tidb-operator"),
+		})
+	if err != nil {
+		return nil
+	}
+
+	fmt.Fprintf(o, "Client Version: %s\n", clientVersion.GitVersion)
```
Comment (onlymellb) [inline pkg/tkctl/cmd/version/version.go]: Use `clientVersion` is enough

Comment (aylei) [reply pkg/tkctl/cmd/version/version.go]: Brilliant, you have all the codes in your brain I suppose...

**Comment by aylei** [comment]
@tennix @weekface Thanks for your review, I've made some updates since then, could you PTAL again?

**Pull request merged**